src/main/pages/che-7/overview/proc_installing-che-on-azure-using-the-chectl-command.adoc

```diff
@@ -8,7 +8,7 @@
 
 .Prerequisites
 
-* The `{prod-cli}` management tool is installed. See link:{site-baseurl}che-7/installing-the-{prod-cli}-management-tool/[Installing the `{prod-cli}` management tool].
+* The `{prod-cli}` management tool is installed. See link:{site-baseurl}che-7/using-the-chectl-management-tool/[Installing the `{prod-cli}` management tool].
```

Comment:
Is it preferable to use template for chectl, Che, etc everywhere?

Reply:
Yes. The purpose is to be friendly with downstream project using a different name for the CLI.

Please note also that all links to URLS will be converted into xrefs to file names in a few days when we finalize the migration to Antora.